schemas/yazi.json

```diff
@@ -223,6 +225,42 @@
 				}
 			}
 		},
+		"confirm": {
+			"type": "object",
+			"properties": {
+				"trash_title": { "type": "string" },
+				"trash_origin": {
+					 "$ref": "#/$defs/origin"
+				},
+				"trash_offset": { 
+					"$ref": "#/$defs/offset" 
+				},
+				"delete_title": { "type": "string" },
+				"delete_origin": { 
+					"$ref": "#/$defs/origin"
+				},
+				"delete_offset": { 
+					"$ref": "#/$defs/offset" 
+				},
+				"overwrite_title": { "type": "string" },
+				"overwrite_content": { "type": "string" },
+				"overwrite_origin": {
+					"$ref": "#/$defs/origin" 
+				},
+				"overwrite_offset": { 
+					"$ref": "#/$defs/offset"
+			  },
+				"quit_title": { "type": "string" },
+				"quit_content": { "type": "string" },
+				"quit_origin": { 
+					"$ref": "#/$defs/origin" 
+				},
+				"quit_offset": { 
+					"$ref": "#/$defs/offset" 
+				}
+			},
+			"additionalProperties": false
+		},
```

Comment:
You missed removing these properties from the old `input` section as well: https://github.com/sxyazi/yazi/commit/92112de1c05c288c631f669523fc40fd4eb709d2#diff-86ad26ab85d77e4bd594a4813c46c5292a93e9ddfc9f4c224e3b9d8627635b74.

Reply:
No worries. Thanks!